src/losses/functions.jl

```diff
@@ -503,3 +503,32 @@ end
 ```@meta
 DocTestFilters = nothing
 ```
+
+"""
+  margin_ranking_loss(x1, x2, y; margin=0, mode=identity)
```

Comment:
```suggestion
    margin_ranking_loss(x1, x2, y; margin=0, mode=identity)

```
Also please add the definition to the docstring as other losses do

Reply:
Btw it should be updated with agg instead of mode